src/index.html

```diff
@@ -5,10 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#home"  class="header__icon-link">
+          <img src="./images/logo.png" alt="moyo_icon">
+        </a>
+      </div>
```

Comment:
Still redundant `div` here - please address @Moroz-Dmytro comment

Hint - `a` is also a container component - in your case you just can add all classes to `a` and throw away the `<div class="header__icon">`